docs/api/app-management.md

```diff
@@ -26,3 +26,4 @@ View and install Sumo Logic applications that deliver out-of-the-box dashboards,
 | JP         | https://api.jp.sumologic.com/docs/#tag/appManagement  |
 | US1        | https://api.sumologic.com/docs/#tag/appManagement     |
 | US2        | https://api.us2.sumologic.com/docs/#tag/appManagement |
+| KR        | https://api.kr.sumologic.com/docs/#tag/appManagementV1 |
```

Comment:
indent is one character off in a few places like this

Reply:
No issues with the indent, the published page will look fine.